hardware/OTGWBase.cpp

```diff
@@ -399,7 +400,7 @@ void OTGWBase::ParseLine()
 		_status.Return_water_temperature = static_cast<float>(atof(results[idx++].c_str()));				SendTempSensor(idx - 1, 255, _status.Return_water_temperature, "Return Water Temperature");
 		_status.DHW_setpoint_boundaries=results[idx++];
 		_status.Max_CH_setpoint_boundaries=results[idx++];
-		_status.DHW_setpoint = static_cast<float>(atof(results[idx++].c_str()));							UpdateSetPointSensor(idx - 1, _status.DHW_setpoint, "DHW Setpoint");
+		_status.DHW_setpoint = static_cast<float>(atof(results[idx++].c_str())); if(_status.DHW_setpoint!=0.0f)							UpdateSetPointSensor(idx - 1, _status.DHW_setpoint, "DHW Setpoint");
```

Comment:
This if should be on a separate line please
Maybe this patch needs to be implemented for the others as well (return water, control setpoint, boiler water)